src/test/java/org/folio/rest/support/kafka/FakeKafkaConsumer.java

```diff
@@ -50,6 +57,15 @@ public FakeKafkaConsumer consume(Vertx vertx) {
     consumer.handler(message -> {
       final List<KafkaConsumerRecord<String, JsonObject>> storageList;
 
+      // Messages that are earlier than the timestamp are stale and should be ignored.
+      if (message.timestamp() < timestamp.toInstant().getMillis()) {
+        logger.debug("Timestamp is " + message.timestamp()
+          + " less than " + timestamp.toInstant().getMillis()
+          + " for message: " + message.topic() + ", " + message.key() + ", "
+          + message.value().encodePrettily());
+        return;
+      }
```

Comment:
I think this drops messages that have been received out of order based upon the timestamp of the messages.

Have I understood that correctly?

Reply:
Yes this is the idea.
My understanding is that this is using Kafkas timestamp rather than any Folio crafted one.
However, the fact that logs do not appear when debug is enabled has shown that this is not where the problem happens.

I left this debug logger because it still may come in handy should this problem ever actually occur.
Ideally we should never see this log message appear when debug is enabled.

Reply:
I think we need to be careful with this. 

I don't know the source of the timestamp, however if it is in the message, it probably comes from the publisher.

I also don't think we can rely on only wanting messages received in order.

Reply:
> I think we need to be careful with this.

I can remove the return statement and leave the log message just in case.